sdk/src/main/scala/io/horizen/account/state/Invocation.scala

```diff
@@ -55,10 +57,23 @@ case class Invocation(
     } else if (readOnly) {
       TracerOpCode.STATICCALL
     } else {
-      TracerOpCode.CREATE
+      TracerOpCode.CALL
     }
   }
 
+  override def toString: String =
```

Comment:
Is this method currently used in the SDK implementation?

Reply:
I don't know but it is possible in some logging.